src/renderer/components/widgets/channels/BaseChannelsList.tsx

```diff
@@ -50,20 +44,4 @@ export const BaseChannelsList = ({
   )
 }
 
-BaseChannelsList.propTypes = {
-  channels: PropTypes.array.isRequired,
-  unknownMessages: PropTypes.array.isRequired,
-  selected: PropTypes.object.isRequired,
-  selectedOffer: PropTypes.object,
-  directMessages: PropTypes.bool
-}
-
-BaseChannelsList.defaultProps = {
```

Comment:
Leave default values for things like channels and unonown messages. This way you don't have them optional in props.